examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -13,8 +13,7 @@
    "source": [
```

Comment:
<p>The next cell has nothing to do with <code>default_config</code>,  and you don't really need to know about its existence.</p><p>Describe what <code>layout</code>  is, and what different letters stand for</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>